src/shared_modules/utils/rocksDBQueue.hpp

```diff
@@ -108,7 +137,10 @@ class RocksDBQueue final
 
         while (it->Valid())
         {
-            auto key = std::stoull(it->key().ToString());
+            auto keyString = it->key().ToString();
+            auto key = std::stoull(keyString);
+            keyNormalization(key, keyString, it);
```

Comment:
This normalization can be expensive if the environment has a lot of queued events (we have instances in the order of the millions)

We are going to initialize the module in legacy or fast mode depending on the following:
- Are there more than 1000 remaining keys without padding? -> Legacy mode
- Are there less than 1000 remaining keys without padding? -> Apply normalization and use fast mode

> [!NOTE]
> If at least one legacy key is present, all remaining keys will be legacy

Reply:
Added!